lib/data_pull.rb

```diff
@@ -263,4 +266,46 @@ def run(args:, config:)
       )
     end
   end
+
+  class ProfileSummary
+    def run(args:, config:)
+      uuids = args
+
+      users = User.includes(:profiles).where(uuid: uuids).order(:uuid)
+
+      table = []
+      table << %w[uuid current_status activated_timestamp disabled_reason fraud_review_pending_timestamp fraud_rejection_timestamp]
+
+      users.each do |user|
+        if user.profiles.any?
+          user.profiles.sort_by(&:id).each do |profile|
+            profile_status = profile.active ? 'Active' : 'Inactive'
+            table << [
+              user.uuid,
+              profile_status,
+              profile.activated_at,
+              profile.deactivation_reason,
+              profile.fraud_review_pending_at,
+              profile.fraud_rejection_at,
+            ]
+          end
+        elsif config.include_missing?
+          table << [user.uuid, '[HAS NO PROFILE]', nil, nil, nil, nil]
```

Comment:
I think that we can skip the rest of the `nil` values here, since the table gem and CSV stuff should just render correctly
```suggestion
          table << [user.uuid, '[HAS NO PROFILE]']
```

Reply:
![image](https://github.com/18F/identity-idp/assets/109746710/211a272f-fb6b-4777-a032-d1a64fde1bb3)
If I remove "nil" table display like this

Reply:
oh bummer! I hoped the gem would be smarter than that 😦